README.md

```diff
@@ -147,26 +147,26 @@ Don't see the service you are looking for? We are constantly adding support for
 
 ## Concepts
 
-- **User** - A person using your web application. This could be you, your friends, or your customers.
-- **Tenant** - A large multi-user system uses a concept of a _tenant_ to identify the larger organization a particular _user_ may belong to. For example, your _user_ might be Janet, but the _tenant_ might be Sonicity, a large multinational corporation. Generally, _users_ authenticate on behalf of _tenants_, though for single-user environments the _user_ and the _tenant_ might be effectively the same.
+- **User** - A person using your web application; it could be you, your friends, or your customers.
+- **Tenant** - A large multi-user system uses a _tenant_ concept to identify the larger organization to which a particular _user_ may belong. For example, your _user_ might be Janet, but the _tenant_ might be Sonicity, a large multinational corporation. Generally, _users_ authenticate on behalf of _tenants_, though for single-user environments the _user_ and the _tenant_ might be effectively the same.
 - **Service** - A service is a remote SaaS your users are already using, upon which you would like to act on their behalf. For example, your application may modify a HubSpot record, send a message to Slack, or update a Salesforce company on behalf of your application users.
 - **Identity** - The necessary tokens, refresh tokens, or other secrets that are used to authorize API calls to a _service_ on behalf of a _user_.
-- **Tag** - A key-value pair, for example ("userId", "user-123"). A number of tags can be associated with an _identity_. EveryAuth enables you to look up an _identity_ or _identities_ that are associated wth a specific set of _tags_.
+- **Tag** - A key-value pair, for example ("userId", "user-123"). A number of tags can be associated with an _identity_. EveryAuth enables you to lookup an _identity_ or _identities_ associated with a specific set of _tags_.
 
 ## Authentication
 
-EveryAuth CLI and middleware communicate with the Fusebit APIs to do their job, and need to authorize those calls. The credentials to do so are established when you run `everyauth init` and stored in the `.fusebit/settings.json` file on disk.
+EveryAuth CLI and middleware communicate with the Fusebit APIs to do their job and need to authorize those calls. The credentials are established when you run `everyauth init` and stored in the `.fusebit/settings.json` file on disk.
 
-Both the CLI and express middleware locate credentials in a similar manner, in priorty order:
+Both the CLI and express middleware similarly locate credentials, in priority order:
```

Comment:
Actually, the CLI looks for them only on the filesystem in ~/.fusebit.

It's just the express middleware that does the search action.